ci/ci/pipeline.yml

```diff
@@ -1548,6 +1548,7 @@ jobs:
           ENV_FILE: foundations/config/env.yml
           CONFIG_FILE: ci/config/runtime-config.yml
           NAME: my-runtime-config
+          BOSH_ENV_PREFIX: OM_
```

Comment:
Is the trailing `_` required? I thought we saw that it was added automatically when we were testing locally?

Reply:
I removed the _  and added a new commit message.  Can you please review?